doc/source/howtos/upgrade_guide.rst

```diff
@@ -211,14 +205,14 @@ upgraded offline for an extended period. More on this below.
 When performing any upgrade (major or minor), :term:`SST` could
 be initiated by the joiner node after the upgrade if the server
 was offline for some time. After :term:`SST` completes, the data
-directory structure needs to be upgraded (using mysql_upgrade)
-once more time to ensure compatibility with the newer version
+directory structure needs to be upgraded
```

Comment:
This section can be removed (or maybe modified to mention that manual mysql_upgrade is not needed).  The SST process will run mysql_upgrade if necessary.  So at the end of the SST, the datadir should be upgraded to the current version (if necessary).  This is true for both minor/major version upgrades.